matrix/commands.py

```diff
@@ -544,6 +544,15 @@ def hook_commands():
         hook_page_up()
 
 
+def hook_key_bindings():
+    W.hook_hsignal("matrix_cursor_reply", "matrix_cursor_reply_signal_cb", "")
+
+    binding = "@chat(python.{}*):r".format(G.BUFFER_NAME_PREFIX)
```

Comment:
If the buffer names always start with matrix (seems that way from #220), you might want to make the binding only for those buffers. wee-slack has an option for dropping the slack buffer name prefix, which is why we couldn't do that.

I.e.
```
"@chat(python.matrix.{}*):r".format(G.BUFFER_NAME_PREFIX)
```

Reply:
In the first commit for this PR, `BUFFER_NAME_PREFIX` [is defined](https://github.com/poljar/weechat-matrix/pull/221/commits/de1d3d466451969993b73d08ab547dc8ebecb8f2#diff-2a51ee9b273943a3328a57e1b1125782R45) to be `matrix.`, so the code as-is should work out precisely to what you are suggesting. :)

Reply:
Perhaps what you're really saying is that `BUFFER_NAME_PREFIX` is a bad name for that constant. If so, then I don't really know what would be a better name; suggestions?

Reply:
Ah, right, of course. Not sure how I managed to miss that :) `BUFFER_NAME_PREFIX` is a good name, I just didn't think about it being included here for some reason (even though it clearly is in the code I commented on, and pasted).